kola/tests/misc/gce.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2018 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package misc
+
+import (
+	"encoding/json"
+
+	"github.com/coreos/go-semver/semver"
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/kola/register"
+)
+
+func init() {
+	register.Register(&register.Test{
+		Name:        "coreos.misc.gce.agent",
+		Platforms:   []string{"gce"},
+		Run:         gceVerifyAgentIsRunning,
+		ClusterSize: 1,
+		// Previously broken on NVMe devices, see
+		// https://github.com/coreos/bugs/issues/2399
+		MinVersion: semver.Version{Major: 1828},
```

Comment:
There's no need to carry a `MinVersion` with a major older than the current stable.